.github/workflows/haskell.yml

```diff
@@ -12,13 +12,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ["8.10.7"]
-        os: [ubuntu-latest, macos-latest, windows-latest]
-        aeson: ["2.0.3.0"]
-        include:
-          - ghc: "8.10.7"
-            os: ubuntu-latest
-            aeson: "1.5.6.0"
+        ghc: ["8.10.7", "9.2.8", "9.6.5"]
```

Comment:
We may want to consider dropping 9.2.8?

Reply:
Should we move this out to an issue after getting this merged or is it something we should address prior to proceeding with next steps of getting the rest of the build processes back to a working state @disassembler?